packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/AccordionDemo/AccordionDemo.tsx

```diff
@@ -10,6 +10,10 @@ export class AccordionDemo extends React.Component<null, AccordionDemoState> {
     expanded: null
   };
 
+  componentDidMount() {
+    window.scrollTo(0, 0)
```

Comment:
Whenever I click on a link in the left-side navigation pane, this code jumps to the top of the page. It forces me to scroll to the next link to see another example.

Ideally, the main content area would be scrollable, separate from the navigation pane, so users don't need to jump around so much. I believe that is beyond the scope of this PR and should be addressed separately,

Reply:
@dlabrecq Thanks for this feedback! I checked with @redallen about this before I added the `componentDidMount()`, and he thought it would be okay for me to include. Otherwise, the alternative is being forced to always scroll up to the top of the page each time you click on a new demo in the sidebar. Either way, I see how it can get annoying 🙂

 Do you think it would be okay to keep this functionality for now, and then let the scrollable content area be addressed in a PR that's out of this scope?

Reply:
Yea, no worries. Next time...